### PR TITLE
Added all HTTP Method verbs

### PR DESCRIPTION
**Pull request by CharlesOkwuagwu**

### CODE REVIEW

**Comment by CrowdHailer** [comment]
This looks a sensible addition

File: lib/ace/http2.ex [hunk 1]
```diff
@@ -13,7 +13,8 @@ defmodule Ace.HTTP2 do
 
   import Kernel, except: [send: 2]
 
-  @known_methods ["GET", "POST", "PUT", "PATCH", "DELETE", "HEAD"]
+  # @known_methods ["GET", "POST", "PUT", "PATCH", "DELETE", "HEAD"]
```
Comment (CrowdHailer) [inline lib/ace/http2.ex]: This commented line is not necessary

Comment (CharlesOkwuagwu) [reply lib/ace/http2.ex]: ok

**Comment by lpil** [comment]
Looks good to merge to me :)

**Comment by CrowdHailer** [comment]
will get onit. leaving as a reminder because I need time to make a release and add comment to CHANGELOG

**Pull request merged**